onnx/onnx-ml.proto

```diff
@@ -592,8 +596,4 @@ message OperatorSetIdProto {
   // The version of the operator set being identified.
   // This field MUST be present in this version of the IR.
   optional int64 version = 2;
-}
-
-// For using protobuf-lite
-option optimize_for = LITE_RUNTIME;
-
+}
```

Comment:
endline charactor?

Reply:
This was copied as one of the build products. I will re-do change manually.

---
In reply to: [344909908](https://github.com/onnx/onnx/pull/2437#discussion_r344909908) [](ancestors = 344909908)